<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mLabValueExtractor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="grp_inputFile">
      <property name="title">
       <string>select data file path</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_selectInputFile">
         <property name="text">
          <string>select data file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_inputFile">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="grp_options">
      <property name="title">
       <string>data options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="3" colspan="5">
        <widget class="QLineEdit" name="txt_firstValue">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>e.g.: counter: </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QSpinBox" name="spb_startValue">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="5">
        <widget class="QLineEdit" name="txt_secondValue">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QSpinBox" name="spb_stopValue">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item row="4" column="4" colspan="2">
        <widget class="QLabel" name="lbl_valueRangeTo">
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QLabel" name="lbl_valueRangeExplanation">
         <property name="text">
          <string>(0: all from start)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="5">
        <widget class="QComboBox" name="cob_delimiter"/>
       </item>
       <item row="0" column="5" colspan="2">
        <widget class="QRadioButton" name="rad_twoValues">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>2 values</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QRadioButton" name="rad_oneValue">
         <property name="text">
          <string>1 value + counter</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="lbl_firstValue_lbl">
         <property name="text">
          <string>first value:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QRadioButton" name="rad_allValues">
         <property name="text">
          <string>all values</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="4">
        <widget class="QRadioButton" name="rad_inRange">
         <property name="text">
          <string>values in range:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="lbl_secondValue">
         <property name="text">
          <string>second value:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_valueRangeFrom">
         <property name="text">
          <string>from</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lbl_delimiter_lbl">
         <property name="text">
          <string>delimiter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QComboBox" name="cob_pointOrKomma"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="grp_outputFile">
      <property name="title">
       <string>select data file path</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_selectOutputFile">
         <property name="text">
          <string>select output file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_outputFile">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="btn_extract">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>get values!</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTextEdit" name="txt_info">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn_clearInfo">
      <property name="text">
       <string>clear info</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_exit">
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
